README.md

```diff
@@ -0,0 +1,97 @@
+# HTML & CSS3 Course
```

Comment:
It's good that you included readme file, but kindly make sure that your readme file properly explains what you did on the project. considering this I will highly encourage you to re-write the readme file.